wp-content/themes/core/assets/pcss/global/reset.pcss

```diff
@@ -0,0 +1,76 @@
+/* -------------------------------------------------------------------------
+ *
+ * Global "Resets" & Light Normalize
+ *
+ * ------------------------------------------------------------------------- */
+
+/**
+ * Use a more-intuitive box-sizing model.
+ */
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+/**
+ * Prevent adjustments of font size after orientation changes in iOS.
+ */
+html {
+	text-size-adjust: 100%;
+}
+
+/**
+ * 1. Improve text rendering.
+ * 2. Prevent horizontal scroll.
+ */
+body {
+	-webkit-font-smoothing: antialiased;
+	overflow-x: hidden;
+}
+
+/**
+ * Avoid text overflows.
```

Comment:
If we're really trying to keep things minimal, this is one thing I debated adding. Open to thoughts/opinions on this.

Reply:
Hmm... Yeah, I can see the hesitation. I think it's fine to add though. We end up adding it piece-meal to most project sat some point anyway.  One suggestion: can you use [the `--heading` custom selector](https://github.com/moderntribe/moose/blob/main/wp-content/themes/core/assets/pcss/custom-selectors/_variables.pcss#L13) instead of calling out all headings individually?

Reply:
Yes, I need to get used to those! Done.